tools/get_lib_manpages.lua

```diff
@@ -0,0 +1,63 @@
+--[[
```

Comment:
I figured that I would give you a full solution that you could use rather than writing half a solution and making someone else come up with the other half.  That way you could use what you have, build on what I built or just use it as is.  I figured you might be a little busy ;)